cmd/helm/install.go

```diff
@@ -119,7 +119,7 @@ func (i *installCmd) run() error {
 		helm.ValueOverrides(rawVals),
 		helm.ReleaseName(i.name),
 		helm.InstallDryRun(i.dryRun),
-		helm.InstallReuseName(i.reuseName),
+		helm.InstallReuseName(i.replace),
```

Comment:
My think on that was this: 

- From the Helm CLI perspective, this flag is doing a replacement
- On the server side, we're just controlling whether or not a name is re-usable

Today, these equate to the same thing, but I'm not sure that that will be true going forward. I could imagine wrapping more logic around `--replace` in addition to just reusing the same name.

I kinda don't want to make a single boolean flag called `Replace` on the protobuf object, because that feels like a really high-level operation, while `ReuseName` feels like a really specific operation.

Does that make sense? Am I over-architecting this?